spec/classes/server_spec.rb

```diff
@@ -31,7 +31,7 @@
             :servertype   => server_type,
             :storeconfigs => 'puppetdb',
             :environmentpath => '$confdir/environments',
-            :basemodulepath => '/etc/puppet/modules:/usr/share/puppet/modules',
+            :basemodulepath => ['/etc/puppet/modules', '/usr/share/puppet/modules'],
```

Comment:
seems excessive, and an unnecessary break of backwards compatibility.
going forward, in [puppet 4](http://puppet-on-the-edge.blogspot.se/2013/12/variant-data-and-type-and-bit-of-type.html) i'd simply make this
```puppet
  Variant[String, Array[String]] $basemodulepath = undef,
```